Source/Node/src/MFRC522Reader.cpp

```diff
@@ -0,0 +1,112 @@
+#include "MFRC522Reader.hpp"
+#include <SPI.h>
+
+
+#define DEFAULT_SS_PIN 10
+#define DEFAULT_RST_PIN 9
+#define DEFAULT_KEY 0xFF
+#define DEFAULT_TRAILER_BLOCK 7
+
+MFRC522Reader::MFRC522Reader()
+{
+    reader = MFRC522(DEFAULT_SS_PIN, DEFAULT_RST_PIN);
+    SPI.begin();
+    reader.PCD_Init();
+    for (byte i = 0; i < 6; i++)
+    {
+        key.keyByte[i] = 0xFF;
+    }
+}
+
+MFRC522Reader::MFRC522Reader(int SS_PIN, int RST_PIN)
+{
+    reader = MFRC522(SS_PIN, RST_PIN);
+    SPI.begin();
+    reader.PCD_Init();
+    for (byte i = 0; i < 6; i++)
+    {
+        key.keyByte[i] = 0xFF;
```

Comment:
More mentions of 0xFF, but it was defined above.

Also, what is that 6 doing there?